<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientSignUpDialog</class>
 <widget class="QDialog" name="ClientSignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FF6200, stop:1 #00CC66);
    color: #FFFFFF;
    border-radius: 10px;
    border: none;
    padding: 5px;
    font-family: &quot;Poppins&quot;;
    font-size: 16pt;
    font-weight: 600;
    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #00CC66, stop:1 #FF6200);
    border: 2px solid #535AEF;
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #CC4E00, stop:1 #00994C);
    border: 2px solid #4488CC;
}</string>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="surnameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="confirmPasswordLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneLineEdit"/>
    </item>
    <item>
     <widget class="QDateEdit" name="birthDateEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="adressLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
